ui/src/app/components/apps-panel.spec.tsx

```diff
@@ -64,7 +64,7 @@ const findAvailableApps = (wrapper: ReactWrapper, activeAppsExist: boolean) => {
 };
 
 const findUnexpandedApp = (wrapper: ReactWrapper, appName: string) =>
-  wrapper.find({ 'data-test-id': `${appName}-unexpanded` });
+  wrapper.find({ 'data-test-id': `${appName}-unexpanded` }).at(0);
```

Comment:
The `.at(0)`s are caused by `propagateDataTestId` passing `data-test-id` to several components. This is how other components `propagateDataTestId` using deal with this.